### ISSUE TITLE
Making this part of qutebrowser's docs?

### ISSUE THREAD

**Issue by The-Compiler** [lufte/awesome-qutebrowser-userscripts#1]
Hey - just found this repository!

There's currently a qutebrowser PR open which adds a README to the userscript folder (where most userscripts live), to link to userscripts which are hosted elsewhere:

https://github.com/sim590/qutebrowser/blob/bd1cdd6b27b1de3f122e580957482b2f0afc2919/misc/userscripts/README.md
https://github.com/qutebrowser/qutebrowser/pull/4029

I'm currently mostly ignoring PRs (as exams are coming up), but once that's in, I'd much rather prefer links to be added there rather than in a separate repository - it's probably easier to find there. In the long run (after https://github.com/qutebrowser/qutebrowser/issues/345) that could even become a proper part of qutebrowser's docs.

What do you think?

**Comment by lufte** [comment]
Hey, I just made it, so that was pretty fast.

Sure, that makes a lot more sense. So anyone that wants to "publish" their userscript will simply send you a pr to modify that file?

I guess I'll delete the repo then, as the list barely contains just one script I wrote.

Thanks for getting in touch.

**Comment by The-Compiler** [comment]
Yeah, that's how it'd work. I guess I'd add the others [you've found](https://github.com/lufte) as well - it's nice that you asked for permission to do so, but I don't feel like it'd be strictly necessary :wink:
